addons/packages/vsphere-csi/2.4.1/package.yaml

```
@@ -5,21 +5,118 @@ metadata:
 spec:
   refName: vsphere-csi.community.tanzu.vmware.com
   version: 2.4.1
-  releaseNotes: "vsphere-csi 2.4.1-rc.1 https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.4.1-rc.1"
+  releaseNotes: vsphere-csi 2.4.1-rc.1 https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.4.1-rc.1
   licenses:
-    - "Apache 2.0"
+  - Apache 2.0
   template:
     spec:
       fetch:
-        - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/vsphere-csi@sha256:c615f4a95de0161f244f644cfb512449ef8b2c233e63541d16fa44bcd7bd28b6
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/vsphere-csi@sha256:ffb6731278dc157d75ff56e8a4e8f6b5f8f08cc108a86e42c0cb8204ae7203ef
```

```suggestion
          image: projects.registry.vmware.com/tce/vsphere-csi@sha256:ed2b01baf6e93a5a7bfb7d79f3602735ab4cd3ab0e3a68c9856a64e6894a3ae3
```
Locally I rebased with main and pushed the package, this is the SHA that I got.